tests/unit/plugins/inventory/test_digitalocean.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2021 Ansible Project
+# GNGeneral Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import pytest
+
+from ansible.errors import AnsibleParserError
+from ansible.inventory.data import InventoryData
+from ansible_collections.community.digitalocean.plugins.inventory.digitalocean import InventoryModule
+
+
+@pytest.fixture(scope="module")
```

Comment:
Is there a reason for scoping this fixture to a test module? This feels unsafe since inventory plugins are not "pure" and do have mutable internal states. I would suggest that we leave out the scope parameter and just use the default function scope that will deliver a fresh instance on each test function invocation.

Reply:
I guess this comes from some of the existing tests (I've seen it there as well). I agree that not specifying `scope="module"` is better.

Reply:
er... yes, I copied what I found without knowing what it does... deleted and still works :)